forge/routes/api/application.js

```diff
@@ -276,9 +276,49 @@ module.exports = async function (app) {
         }
     })
 
+    /**
+     * Get a list of devices owned by this application
+     * @name /api/v1/applications/:id/devices
+     * @static
+     * @memberof forge.routes.api.application
+     */
+    app.get('/:applicationId/devices', {
+        preHandler: app.needsPermission('team:device:list'),
+        schema: {
+            summary: 'Get a list of all devices in an application',
+            tags: ['Application Devices'],
```

Comment:
```suggestion
            tags: ['Applications'],
```

Not sure a separate section is needed - keeping it consistent with the /instances end point for eg

Reply:
Should this then not be `['Devices']` as it returns devices?

Not sure now that I understood the relevance of tags 🤔

Reply:
The tags determine where it shows up in the swagger doc output - https://app.flowforge.com/api/static/index.html

This is an application based API so should be under applications.